topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,347 @@
+                                 
+9. **analysis_method** [ Default: "LM" ] [ Options: "LM", "CPLM", "ZICP", "NEGBIN", "ZINB" ]
+- The analysis method to apply.\
+- Options: \
+        1. <u>Linear Model (LM)</u>: Determines how changes in metadata are associated with changes in the taxonomy data. 
+        2. <u>Compositional Proportional Linear Model (CPLM)</u>: used for analyzing compositional data, where the taxa abundances are proportions or percentages that sum to 1.
+        3. <u>Zero-Inflated Count Model (ZCIP)</u>: used when there are many zero counts in the microbiome data. It handles datasets where a large number of taxa are absent in many samples. 
+        4. <u>Negative Binomial Model (NEGBIN)</u>: used for count data where there is overdispersion (variance exceeds the mean).
+        5. <u>Zero-Inflated Negative Binomial Model (ZIND)</u>: combines features of both zero-inflation and negative binomial models, useful for count data with both excess zeros and overdispersion.   
+                                                          
+10. **correction or adjustment methods** [ Default: "BH" ] : 
+- When performing numerous statistical tests simultaneously, like testing the association of many microbial taxa with various metadata variables, the risk of finding false positives increases. 
+- Correction methods help control this risk to ensure that the results are reliable and that significant findings are not due to random chance.
+- This is done by computing the q-value, which is a measure of how many false positives are expected among the significant results. 
+- Options:\
+      1. <u>Benjamini & Hochberg(BH)(aka false discovery rate(fdr))</u>: A common method used for FDR correction. It ranks the p-values from smallest to largest and adjusts them based on their rank and the total number of tests.
+      2. <u>Benjamini & Yekutieli(BY)</u>: Similar to Benjamini-Hochberg but includes a correction factor that accounts for the correlation between tests.
+      3. <u> Bonferroni correction</u>: Divides the significance threshold (alpha level) by the number of tests performed and then compares each p-value to this adjusted significance level to determine if it is statistically significant. 
+      4. <u>Hochberg </u>:  It is similar to the Bonferroni correction but is often more powerful, meaning it has better statistical power to detect true effects while controlling for false positives.
+      5. <u>Hommel</u>: controls the Family-Wise Error Rate (FWER) by adjusting p-values in a step-down fashion, starting from the smallest p-value and progressively increasing the threshold. It is more powerful than Bonferroni while maintaining strict error control.
+      6. <u>Holm</u>:  controls the Family-Wise Error Rate (FWER) by sequentially adjusting p-values from smallest to largest, comparing ch step. This stepwise approach is less conservative than the Bonferroni correction, offering greater statistical power.\
```

Comment:
What is this ` ch step`?

Reply:
its the adjusted critical value at step i,  calculated as α/(m-i+1) ​. Sorry for the confusion, I'll explain it more clearly in the tutorial.